src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -41,27 +43,72 @@ const useStyles = makeStyles(() => ({
     transform: 'rotate(180deg)',
   },
   subMenu: {
-    backgroundColor: theme.palette.cruGrayDark.main,
+    backgroundImage: `linear-gradient(0deg, ${theme.palette.cruGrayDark.main}, ${theme.palette.cruGrayDark.main})`,
   },
   menuItem: {
     color: 'white',
     '&:hover': {
       backgroundColor: theme.palette.cruGrayMedium.main,
+      backgroundBlendMode: 'multiply',
     },
   },
   menuItemSelected: {
+    backgroundBlendMode: 'multiply',
     backgroundColor: theme.palette.cruGrayMedium.main,
   },
+  needsAttention: {
+    backgroundImage: `linear-gradient(0deg, ${theme.palette.mpdxYellow.main}, ${theme.palette.mpdxYellow.main})`,
+  },
   menuIcon: {
     marginRight: theme.spacing(1),
   },
+  notificationBox: {
+    backgroundColor: theme.palette.progressBarYellow.main,
+    paddingLeft: theme.spacing(1),
+    paddingRight: theme.spacing(1),
+    borderRadius: '25%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginLeft: theme.spacing(2),
+  },
+  darkText: {
+    color: theme.palette.cruGrayDark.main,
+  },
+  whiteText: {
+    color: 'white',
+  },
 }));
 
+interface query {
+  totalCount: number;
+}
+
+interface responseData {
+  [key: string]: query;
+}
+
 const NavMenu = (): ReactElement => {
   const { t } = useTranslation();
   const classes = useStyles();
   const accountListId = useAccountListId();
   const currentToolId = useCurrentToolId();
+  const { data, loading } = useGetToolNotificationsQuery({
+    variables: { accountListId: accountListId ?? '' },
+  });
+  const [dataState, setDataState] = useState<responseData>({});
```

Comment:
Why is this needed? Can you just use `data` directly?

Reply:
I cant do something like data['fixCommitmentInfo'], well it does work but it gives typescript warnings and idk if we can somehow change the return type to allow it to work